source/common/router/header_formatter.cc

```diff
@@ -143,7 +143,10 @@ RequestInfoHeaderFormatter::RequestInfoHeaderFormatter(absl::string_view field_n
     }
     field_extractor_ = [this, pattern](const Envoy::RequestInfo::RequestInfo& request_info) {
       const auto& formatters = start_time_formatters_.at(pattern);
-      ASSERT(formatters.size() == 1);
+      if (formatters.size() != 1) {
+        throw EnvoyException(
+            fmt::format("multiple start_time values found: {}", formatters.size()));
```

Comment:
@anirudhmurali, thanks for checking on this.

Questions: 

1. Do you think "multiple %START_TIME% patterns..." sounds better? 
2. Is the input here: https://github.com/envoyproxy/envoy/pull/4189/files#diff-084669b1d67ce3df5fbda83bef9354bbR3 causing this?

Reply:
NVM, I tried to patch the `header_formatter` to accept this pattern from the corpus, and it seems it makes sense to have multiple patterns inline as the header value. 

@anirudhmurali, as I said above I can work with you to fix this.

(This fuzzer is really good then. This is my mistake, sorry).